packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -610,20 +616,22 @@ export default class Meeting extends StatelessWebexPlugin {
      */
     this.id = uuid.v4();
     /**
-     * Correlation ID used for network tracking of meeting
+     * Call state used for metrics
      * @instance
-     * @type {String}
+     * @type {CallStateForMetrics}
      * @readonly
      * @public
      * @memberof Meeting
      */
-    if (attrs.correlationId) {
+    this.callStateForMetrics = attrs.callStateForMetrics || {};
+    const correlationId = attrs.correlationId || attrs.callStateForMetrics?.correlationId;
```

Comment:
the fact that we can provide it via 2 ways, will be tricky... and annoying to remember

Reply:
I know it's a bit annoying but it seemed better having aall the call state for metrics bits together and it had to be backwards compatible